src/document.h

```diff
@@ -496,6 +480,22 @@ struct Document {
         return;
     }
 
+#ifdef __WXGTK__
+    wxCustomDataObject *CopyEntireCells(wxString &s) {
+#else
+    wxHTMLDataObject *CopyEntireCells(wxString &s) {
+#endif
+        sys->clipboardcopy = s;
+        wxString html = selected.g->ConvertToText(selected, 0, A_EXPHTMLT, this);
+        #ifdef __WXGTK__
+        wxCustomDataObject *htmlobj = new wxCustomDataObject(wxDF_HTML);

```

Comment:
`auto *htmlobj =` outside the ifdef like the original..  and indent

Reply:
Thanks for your reply. I also left the asterik now out because it is also deduced that it is a pointer type.